web-console/knikubevirt/Create-vm/step-3-storage/wizard-storage.md

```diff
@@ -0,0 +1,59 @@
+# Create Virtual Machine Wizard - Storage Step
+
+The 1st screen of the storage section in the wizard flow will show the disks in a table view. The user can add a disk (this could be either a new disk or attach an existing one).
+
+![add disk default](img/Add-disk-default.png)
+
+Once the user clicks the ‘Add disk’ button, a modal will be shown and the user can choose what kind of disk he/she wishes to add and fill in the details. The default selection will be a blank disk.
+
+![add disk modal](img/Add-disk-modal.png)
```

Comment:
The actions in the modal should be "Add" and "Cancel" to be consistent with other steps like Add NIC.